src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -37,6 +41,8 @@
     private static final String LANGUAGE_PROPERTY = "lang";
     private static final String TEXT_PROPERTY = "text";
 
+    private Set<String> supportedLanguages;
```

Comment:
make static - this will be the same for all(?)

Reply:
Make sense